### DISCUSSION TITLE
[BUG?] DO_NOT_DELETE_SPLIST_TENANTADMIN_AGGREGATED_SITECOLLECTIONS is starting to disappear from M365 tenants

### DISCUSSION THREAD

**Discussion by okms** [pnp/powershell#2617] [General]
### Reporting an Issue or Missing Feature
We're using PnP PowerShell quite extensively, and we've started seeing the DO_NOT_DELETE_SPLIST_TENANTADMIN_AGGREGATED_SITECOLLECTIONS going missing from tenants, apparently without anyone actually having deleted it. 

I am fully aware this isn't something where PnP PowerShell is at fault, but this a heads up in case this is an actual change (intentional or not) being rolled out by Microsoft. 

A consequence of this is that Get-PnPHubSiteChild fails, citing an "unknown error" 

A fix, if required, will have to be implemented in [pnp framework](https://github.com/pnp/pnpframework/blob/d038ad31e1bba2b60dae869f38ad700ebe36efbe/src/lib/PnP.Framework/Extensions/TenantExtensions.cs#L87)

If this is better as discussion feel free to convert it, or just close it if you feel this is noise

**Comment by gautamdsheth** [comment]
Hii @okms, looks like a different issue. Tested in a couple of tenants as well as created a new one. This list exists in each of them, maybe it got deleted by mistake or something else?
Will move it to discussion.

**Comment by fug4zi** [comment]
I have the same issue in my tenant.

'DO_NOT_DELETE_SPLIST_TENANTADMIN_AGGREGATED_SITECOLLECTIONS' does not exist at site with URL 'https://xxxxx-admin.sharepoint.com'.
